bentoml/cli/__init__.py

```diff
@@ -420,6 +421,44 @@ def install_completion(append, shell, path):
                 "install it with `pip install click_completion`"
             )
 
+    @bentoml_cli.command(
```

Comment:
it might make more sense to move this to the `cli/bento.py`, which has all the CLI commands for managing saved bundle?

The line ```bento_service_bundle_path = resolve_bundle_path(bento, pip_installed_bundle_path)``` should be safe to remove as well because it is only required for CLI commands if user `pip install` a saved bundle, e.g. with the quick start example:

```bash
# run the quickstart guide code within the bentoml repository directory
python ./guides/quick-start/main.py

# pip installed the saved bundle
pip install $(bentoml get IrisClassifier:latest -q | jq -r ".uri.uri")

# now the CLI for this saved bundle will be available, it contains CLI commands defined in "create_bento_service_cli"
IrisClassifier run predict  --input="[[1,2,3,4]]"
```

Reply:
Moved to bento.py, thanks